docs/dev/developer-guides/bridging/l1-l2-interop.md

```diff
@@ -1,72 +1,50 @@
+L2 to L1 communication is based on transferring the data as a message, and not on L1 transaction execution. 
 
-In the future, we will also add the ability to "prioritize" L1->L2 transactions, allowing users to speed up the inclusion of their transactions in exchange for paying a higher fee to the operator.
+- Find out [how to send a message from L2 to L1](../../how-to/send-message-l2-l1.md) with zkSync Era.
 
-## Priority mode
-
-If the operator fails to process the needed L1 transactions, the system enters in 'Priority mode'. In this mode, everyone can become an operator by staking tokens. The exact details of the priority mode are still under development and will be described in more detail closer to the mainnet launch.
-
-To reduce risks, the alpha mainnet will start with a mechanism to instantly stop and upgrade the network, which contradicts the purpose of the priority mode. Priority mode will be gradually introduced in the following releases.
-
-## L2 -> L1 messaging
-
-The [L2 -> L1 communication](./l2-l1.md), in contrast to L1 -> L2 communication, is based only on transferring the information, and not on the transaction execution on L1. It is a built-in feature, which is made up of two parts: sending a message from L2 and reading it on L1. The first is implemented as a call to an L2 system smart contract. And the second is implemented on the zkSync L1 smart contract as a getter function.
-
-### Sending messages
+## Priority queue
 
-Each message sent from L2 to L1 contains the sender's address and the message itself. The length of the message can be arbitrarily large, but the longer the message, the more expensive it will be to send. The operator must include all messages for the corresponding merkle root (see next paragraph). Hence, all the messages are publicly available, and one does not have to rely on the operator to reveal them.
+1. All transactions types are now supported by the priority queue. 
 
-### Reading messages
+    In Lite, users may have had their funds locked on an L2 smart contract, and not on their own L2 account. Now, before moving funds to L1, they can execute a transaction on the zkSync Era network which releases the funds from the smart contract so they can move them.
 
-Every message sent can be read on-chain. Moreover, it is possible to prove that a message has been sent in a specific L2 block. To make such proof as cheap as possible for both the user and the operator, we store all messages, for each L2 block, in a merkle tree. Accordingly, any L1 smart contract can consume the message sent by providing proof of inclusion in some L2 block. Proof can be generated based only on the data that the operator sent to the zkSync L1 smart contract. The proof can also be obtained via [the API](../../../api/api.md#zks-getl2tol1msgproof).
+    :::info
+    The withdraw functionality in zkSync Era replaces `FullExit` in Lite.
```

Comment:
(considering my last comment) here's another mention of (zkSync) Lite. I also believe that what is written is incorrect -- both versions have a "withdraw" transaction type (which does exactly the same thing)

Reply:
that also got removed in the last commit